vcluster/_partials/manage/upgrade-version.mdx

```diff
@@ -97,28 +97,28 @@ Replace:
     apiVersion: argoproj.io/v1alpha1
     kind: Application
     metadata:
-      name: $VCLUSTER_NAME
+      name: <VCLUSTER_NAME>
       namespace: argocd
     spec:
       project: default
       source:
         chart: vcluster
         repoURL: https://charts.loft.sh
-        targetRevision: $VCLUSTER_VERSION
+        targetRevision: <VCLUSTER_VERSION>
         helm:
-          releaseName: $VCLUSTER_NAME
+          releaseName: <VCLUSTER_NAME>
           valueFiles:
             - vcluster.yaml
       destination:
         server: https://kubernetes.default.svc
-        namespace: $VCLUSTER_NAMESPACE
+        namespace: <VCLUSTER_NAMESPACE>
     ```
 
     Replace:
-
-    - `$VCLUSTER_NAME` with the name of your vCluster.
-    - `$VCLUSTER_VERSION` with the version of vCluster that you want to upgrade to.
-    - `$VCLUSTER_NAMESPACE` with the namespace that is running the vCluster.
+    
+    - `<VCLUSTER_VERSION>` with the version of vCluster that you want to upgrade to.
```

Comment:
Update the order back to what it was as it specifically goes through each variable as it is found in the code snippet. 

1. name
2. version
3. namespace